tests/e2e_logging_pbft.py

```diff
@@ -34,36 +34,39 @@ def run(args):
         network.add_users(primary, network.initial_users)
         LOG.info("Initial set of users added")
 
-        network.open_network(primary)
+        # network.open_network(primary)
+        script = None
+        result = network.propose(1, primary, script, None, "open_network")
+        network.vote_using_majority(primary, result[1]["id"], False)
         LOG.info("***** Network is now open *****")
 
-        with primary.node_client() as mc:
-            check_commit = infra.ccf.Checker(mc)
-            check = infra.ccf.Checker()
+        # with primary.node_client() as mc:
```

Comment:
has this been left commented out by mistake?

Reply:
Eventually the comments will be removed and we will be able to run this code, but we are not there yet